mkdocs.yml

```diff
@@ -190,14 +190,6 @@ nav:
     - Templating: 'concepts/templating.md'
   - Hub:
     - Introduction to Instructor Hub: 'hub/index.md'
-    - Structured Outputs with Vertex AI: 'hub/vertexai.md'
-    - Structured Outputs with Ollama: 'hub/ollama.md'
-    - Structured Outputs with llama-cpp-python: 'hub/llama-cpp-python.md'
-    - Structured Outputs with Together: 'hub/together.md'
-    - Structured Outputs with Anyscale: 'hub/anyscale.md'
-    - Structured Outputs with Groq: 'hub/groq.md'
-    - Structured Outputs with Mistral: 'hub/mistral.md'
-    - Structured Outputs with Cohere: 'hub/cohere.md'
```

Comment:
create a redirect map for the deleted stuff, using the mkdocs redirect map features